frontend/app/model/PackagePromo.scala

```diff
@@ -0,0 +1,8 @@
+package model
+import com.netaporter.uri.Uri
+
+object PackagePromo {
+
+  // the call-to-action button on the package promo view element
+  case class CtaButton(text: String, to: Uri, attributes: Map[String, String])
```

Comment:
Ah so remembering back I originally had a PackagePromo case class which this was the companion object of, then I scaled back my refactoring since this has a deadline.

I think this would be the place for this class to go if PackagePromo was a case class but since it isn't yet I'm not so sure.

If you're happy for this to be merged I'll spend some time later this week on a PackagePromo case class